src/main/java/br/ufs/hu/prevsep/web/api/dto/form/sepse/PageableDoctorFormDTO.java

```diff
@@ -68,12 +73,17 @@ public void setDtCriacaoEnd(LocalDate dtCriacaoEnd) {
     @Override
     public Predicate getQueryPredicate(QFormularioSepseMedicoEntity qEntity) {
         BooleanBuilder filter = new BooleanBuilder();
+        BooleanBuilder subfilter = new BooleanBuilder();
         if (this.idFormulario != null)
             filter.and(qEntity.idFormulario.eq(this.idFormulario));
         if (this.crmMedico != null)
             filter.and(qEntity.crmMedico.eq(this.crmMedico));
         if (this.status != null)
-            filter.and(qEntity.status.eq(status.getValue()));
+            subfilter = new BooleanBuilder();
+            for(var s: this.status) {
+                subfilter.or(qEntity.status.eq(s.getValue()));
+            }
+            filter.and(subfilter.getValue());
```

Comment:
Essa linha não está dentro do if, e ela só deve executar se o if for true. (colocar os {} do if, que são obrigatórios quando o if tem mais de 1 linha)